aws/modules/privatebastion/module.ftl

```diff
@@ -0,0 +1,91 @@
+[#ftl]
+
+[@addModule
+    name="privatebastion"
+    description="Bastion access via SSM only"
+    provider=AWS_PROVIDER
+    properties=[
+        {
+            "Names" : "tier",
+            "Type" : STRING_TYPE,
+            "Description" : "The tier to use to host the private bastion",
+            "Default" : "msg"
+        },
+        {
+            "Names" : "component",
+            "Type" : STRING_TYPE,
+            "Description" : "The component to use to host the private bastion",
+            "Default" : "ssh"
+        },
+        {
+            "Names" : "deploymentUnit",
+            "Type" : STRING_TYPE,
+            "Description" : "The deployment unit for the private bastion",
+            "Default" : "ssh"
+        },
+        {
+            "Names" : "multiAZ",
+            "Type" : BOOLEAN_TYPE,
+            "Description" : "Multi-AZ support on the private bastion",
+            "Default" : true
+        }
+    ]
+/]
+
+[#macro aws_module_privatebastion
+            tier
+            component
+            deploymentUnit
+            multiAZ ]
+
```

Comment:
If the tier simply must be empty, then perhaps a check here could ensure that? Otherwise you might have someone regenerate/redeploy their VPC only to get to the module `deploymentunit` itself only to run into issues and have to roll-back.